lisa/tools/git.py

```diff
@@ -215,3 +227,8 @@ def get_tag(
             return tags[-1]
         else:
             return tags[0]
+
+    def get_version(self) -> VersionInfo:
```

Comment:
@squirrelsc Would you prefer me to add this functionality to Tool instead of Git? I'm not sure whether `--version` is considered universal to all Linux commands, but this code seems pretty generic.

Reply:
Let's see if there are more requirements. The implementation is not common, even the return value. The VersionInfo need a sematic version string, but some tools don't follow it.